### PR TITLE
Added two Computer Organization and Architecture to Courses in English

### PR DESCRIPTION
**Pull request by moazamdev**
## What does this PR do?
Added two free courses Computer Organization and Architecture section in Courses in English

### Why is this valuable (or not)?
It gives an introduction to computer organization and architecture. This course has everything to know about computer organization and architecture.

### How do we know it's really free?
It's on Youtube and website

### For book lists, is it a book? For course lists, is it a course? etc.
They are courses for course lists

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -275,6 +275,8 @@
 
 ### Computer Organization and Architecture
 
+* [Computer Architecture Complete course](https://youtu.be/9nuAjYRbITQ) - Nerd's lesson (YouTube, Course Divided Into Three Videos)
```
[the remark this answers is not in the document]
Comment (moazamdev) [reply courses/free-courses-en.md:278]: Hi @eshellman, according to your contributing guidelines 

> YouTube videos. We usually don't link to individual YouTube videos unless they are more than an hour long and are structured like a course or a tutorial.

So the link to the course provided by me is structured like a course and it is also more than an hour long.

That's why I added that, but if you ask me to remove it I will remove it.

[the remark this answers is not in the document]
Comment (moazamdev) [reply courses/free-courses-en.md:278]: I have also mentioned the creator and the source of the course

[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-en.md:278]: When the video is long like this, pleas mention that in your description.

[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-en.md:278]: However, you're still using a shortened link

[the remark this answers is not in the document]
Comment (moazamdev) [reply courses/free-courses-en.md]: @eshellman I have added the video length for each video and I also added two more videos of the same course,

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -275,6 +275,10 @@
 
 ### Computer Organization and Architecture
 
+* [Computer Architecture Complete course Part 1](https://youtu.be/9nuAjYRbITQ) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Architecture Complete course Part 2](https://youtu.be/foCkCSZXOt8) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Architecture Complete course Part 3](https://youtu.be/BHMETwOXvWw) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Organization & Architecture](https://www.learncomputerscienceonline.com/computer-organization-and-architecture) - Learn Computer Science
```
Comment (davorpa) [inline courses/free-courses-en.md:281]: Please,

- Don't invent titles. Use the one provided by the original source.
- For YT we prefer playlist URLs. Don't shorten. If are videos, we prefer those > 1h with chapters integrated in timeline.
- Don't invent new notes, e.g. timing length
- Also YT videos/playlist don't need annotations

Fixing...

```suggestion
* [Computer Architecture Complete course Part 1](https://www.youtube.com/watch?v=9nuAjYRbITQ) - Nerd's lesson
* [Computer Architecture Complete Course Part 2](https://www.youtube.com/watch?v=foCkCSZXOt8) - Nerd's lesson
* [Computer Architecture Complete course Part 3](https://www.youtube.com/watch?v=BHMETwOXvWw) - Nerd's lesson
* [Computer Organization And Architecture \| COA Tutorial With Examples](https://www.learncomputerscienceonline.com/computer-organization-and-architecture) - Learn Computer Science
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (moazamdev) [reply courses/free-courses-en.md]: No problem, Actually I am a first-time contributing to open source maybe that's why I didn't understand what you were trying to say.

**Comment by eshellman** [comment]
remove the video length from the listing

**Comment by moazamdev** [comment]
Removed the video length

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -296,9 +296,9 @@
 
 ### Computer Organization and Architecture
 
-* [Computer Architecture Complete course Part 1](https://www.youtube.com/watch?v=9nuAjYRbITQ) - Nerd's lesson (YouTube, Video Length 9:29:50)
-* [Computer Architecture Complete Course Part 2](https://www.youtube.com/watch?v=foCkCSZXOt8) - Nerd's lesson (YouTube, Video Length 9:29:50)
-* [Computer Architecture Complete course Part 3](https://www.youtube.com/watch?v=BHMETwOXvWw) - Nerd's lesson (YouTube, Video Length 9:29:50)
+* [Computer Architecture Complete course Part 1](https://www.youtube.com/watch?v=9nuAjYRbITQ) - Nerd's lesson (YouTube)
+* [Computer Architecture Complete Course Part 2](https://www.youtube.com/watch?v=foCkCSZXOt8) - Nerd's lesson (YouTube)
+* [Computer Architecture Complete course Part 3](https://www.youtube.com/watch?v=BHMETwOXvWw) - Nerd's lesson (YouTube)
```
Comment (davorpa) [inline courses/free-courses-en.md:301]: We don't need YT annotations anymore :wink:

```suggestion
* [Computer Architecture Complete course Part 1](https://www.youtube.com/watch?v=9nuAjYRbITQ) - Nerd's lesson
* [Computer Architecture Complete Course Part 2](https://www.youtube.com/watch?v=foCkCSZXOt8) - Nerd's lesson
* [Computer Architecture Complete course Part 3](https://www.youtube.com/watch?v=BHMETwOXvWw) - Nerd's lesson
```

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
I apologize for not having checked the submitted links sooner, but we do almost a hundred reviews a day during october and can't afford to spend a lot of time on each one.
1. This is an excellent addition.
2. The three videos from Nerd's Lesson are really a single course. Since part 1 links to parts 2 and 3, we don't need links to parts 2 and 3.
3. This is a case where it makes sense to modify the title a bit - you can leave off the "part 1".
4. Nerd's Lesson is unusual for youtube because the course instructors are not part of the channel, which is more of an aggregator than a creator. Here the creator of the course is "David Wentzlaff". Usually we don't fish into the video to get the creators name because they name they want to credit is something like "Code with Harry" or "Codehelp by Babbar". I would list the credit as ` - David Wentzlaff (Nerd's Lesson).

**Review by davorpa — APPROVED** [review]

**Pull request closed without merging**